iOSClient/Networking/NCNetworking+Upload.swift

```diff
@@ -87,12 +88,10 @@ extension NCNetworking {
 
     // MARK: - Upload chunk file in foreground
 
-    @discardableResult
-    @MainActor
-    func uploadChunk(metadata: tableMetadata, hud: NCHud) async -> (account: String,
-                                                                    remainingChunks: [(fileName: String, size: Int64)]?,
-                                                                    file: NKFile?,
-                                                                    error: NKError) {
+    func uploadChunkWithHud(metadata: tableMetadata, hud: NCHud) async -> (account: String,
+                                                                           remainingChunks: [(fileName: String, size: Int64)]?,
+                                                                           file: NKFile?,
+                                                                           error: NKError) {
```

Comment:
Having UI code with business logic is not ideal... but I guess changing this will take a long time now.